pandas/io/common.py

```diff
@@ -183,6 +183,37 @@ def is_gcs_url(url) -> bool:
     return parse_url(url).scheme in ["gcs", "gs"]
 
 
+def _urlopen(*args, **kwargs):
```

Comment:
Should we just make a single `urlopen` here?

Reply:
yes call this urlopen

Reply:
Done.